chainforge/react-server/src/SplitNode.js

```diff
@@ -12,51 +12,11 @@ import { processCSV, deepcopy, deepcopy_and_modify, dict_excluding_key, toStanda
 
 import { fromMarkdown } from "mdast-util-from-markdown";
 import StorageCache from './backend/cache';
-
-const formattingOptions = [
-  {value: "list",    label:"- list items"},
-  {value: "\n",   label:"newline \\n"},
-  {value: "\n\n", label:"double newline \\n\\n"},
-  {value: ",",    label:"commas (,)"},
-  {value: "code",  label:"code blocks"},
-  {value: "paragraph",  label:"paragraphs (md)"},
-];
```

Comment:
These two formatting options (for Join and Split) are actually different constants --this constant is node-specific, and we should keep it here (and not refactor them out.